### PR TITLE
Updated Sirato to Chainlens

### PR DESCRIPTION
**Pull request by NickSneo**
Updated Sirato to Chainlens and some steps to spin up Chainlens free for Besu Developer Quickstart

### CODE REVIEW

File: docs/private-networks/how-to/monitor/chainlens.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+title: Use Chainlens Explorer
+sidebar_position: 7
+description: Use Chainlens Explorer on a privacy-enabled Besu network
+tags:
+  - private networks
+---
+
+# Use Chainlens Blockchain Explorer
+
+[Chainlens Blockchain Explorer](https://www.web3labs.com/chainlens) supports public and private EVM networks. This page describes how to use the free version of Chainlens with its built-in support for [privacy-enabled](../../concepts/privacy/index.md) Besu networks created using the [Developer Quickstart](../../tutorials/quickstart.md).
```
Comment (cfelde) [inline docs/private-networks/how-to/monitor/chainlens.md:11]: We should stick to https://chainlens.com/ anywhere we use https://www.web3labs.com/chainlens

File: docs/private-networks/how-to/monitor/chainlens.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+:::
+
+## Prerequisites
+
+[Docker and Docker Compose](https://docs.docker.com/compose/install/) installed.
+
+## Start Chainlens
+
+Clone the [Chainlens GitHub repository](https://github.com/web3labs/sirato-free):
```
Comment (cfelde) [inline docs/private-networks/how-to/monitor/chainlens.md:27]: The URL needs to change (after renaming repo)

File: docs/private-networks/how-to/monitor/chainlens.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+```bash
+git clone https://github.com/web3labs/sirato-free
```
Comment (cfelde) [inline docs/private-networks/how-to/monitor/chainlens.md:30]: URL needs to change here too

File: docs/private-networks/tutorials/quickstart.md [hunk 4]
```diff
@@ -105,21 +105,21 @@ To display the list of endpoints again, run:
...
+You can [use Chainlens Blockchain Explorer](../how-to/monitor/chainlens.md) to analyze block information, contract metadata, transaction searches, and more. Chainlens has built-in support for privacy-enabled Besu networks.
 
 :::note
 
 You must connect to one of the privacy nodes (for example, `member1besu`), not the dedicated RPC, in order to allow access for Besu [privacy API methods](../reference/api/index.md#priv-methods). In production networks, you must [secure access](../../public-networks/how-to/use-besu-api/authenticate.md) to RPC nodes.
 
 :::
 
-Clone the [Sirato GitHub repository](https://github.com/web3labs/sirato-free):
+Clone the [Chainlens GitHub repository](https://github.com/web3labs/sirato-free):
```
Comment (cfelde) [inline docs/private-networks/tutorials/quickstart.md:116]: URL needs to change here too

File: docs/private-networks/tutorials/quickstart.md [hunk 5]
```diff
@@ -105,21 +105,21 @@ To display the list of endpoints again, run:
...
 
 ```bash
 git clone https://github.com/web3labs/sirato-free
```
Comment (cfelde) [inline docs/private-networks/tutorials/quickstart.md:119]: URL needs to change here too

**Review by cfelde — COMMENTED** [review]
Adding a few reminder comments about changing URL

File: docs/private-networks/how-to/monitor/chainlens.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+The repository contains Docker Compose directory to allow Chainlens to start with a Developer Quickstart test network.
+
+From the docker-compose directory, run the following command:
+
+<!--tabs-->
+
+# Command
+
+```bash
+NODE_ENDPOINT=http://rpcnode:8545 docker-compose -f docker-compose.yml -f sirato-extensions/docker-compose-quorum-dev-quickstart.yml up
```
Comment (cfelde) [inline docs/private-networks/how-to/monitor/chainlens.md:42]: Do we need to update sirato-extensions?

File: docs/private-networks/how-to/monitor/chainlens.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+*screenshots are taken from [Chainlens Goerli network](https://goerli.chainlens.com/dashboard)
+
+The **Dashboard** page provides an aggregated view of network activities.
+
+![`Chainlens_dashboard`](../../../assets/images/chainlens-dashboard.png)
+
+The **Network** page provides an overview of the network status and connected peers. This page is disabled by default, and is only visible if you set `DISPLAY_NETWOR_TAB=enabled` using the following command:
+
+```bash
+NODE_ENDPOINT=http://member1besu:8545 DISPLAY_NETWORK_TAB=enabled docker-compose -f docker-compose.yml -f sirato-extensions/docker-compose-quorum-dev-quickstart.yml up
```
Comment (cfelde) [inline docs/private-networks/how-to/monitor/chainlens.md:75]: Do we need to update sirato-extensions?

**Review by cfelde — COMMENTED** [review]
Found a few remaining references to sirato, please see comments

File: docs/private-networks/how-to/monitor/chainlens.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+This command does two things:
+
+- Sets up the node endpoint
+- Tells Docker to run by using the two Docker Compose files provided
+
+The first `docker-compose.yml` file in the command contains all the services required for Chainlens.
+
+The second file named `docker-compose-besu` contains the network settings required to start
```
Comment (alexandratran) [inline docs/private-networks/how-to/monitor/chainlens.md:53]: Made some content edits to this page. Looks good to me, just need the file in this line to match up with the file in the corresponding example.

File: docs/private-networks/how-to/monitor/chainlens.md (hunk 7 above)
Comment (NickSneo) [reply docs/private-networks/how-to/monitor/chainlens.md]: Changed

File: docs/private-networks/how-to/monitor/chainlens.md (hunk 8 above)
Comment (NickSneo) [reply docs/private-networks/how-to/monitor/chainlens.md]: fixed

**Review by alexandratran — APPROVED** [review]

**Comment by NickSneo** [comment]
@cfelde @alexandratran Can we merge this?

**Comment by alexandratran** [comment]
Looks good to me, let me know if you need me to merge.

**Comment by NickSneo** [comment]
> Looks good to me, let me know if you need me to merge.

Yes please, kindly merge

**Pull request merged**